.circleci/main-config.yml

```diff
@@ -789,38 +790,42 @@ jobs:
         type: string
         default: *pepper_apis_path
     steps:
-      - run:
-          name: Remove directory << parameters.pepper_apis_path >> for checkout of files
-          command: |
-            rmdir << parameters.pepper_apis_path >>
-      - checkout:
-          path: *repo_path
-      - setup-shared-env
-      - run-maven-with-pepper-options:
-          module: dsm-core
-          phase: install
-          skip_tests: true
-          skip_checkstyle: false
-          config_file_path: ''
-          additional_options: -am
-          restore_m2_cache: true
-          save_m2_cache: false
-      - render-configs
-      - create-test-dbs
-      - restore-ddp-cache
-      - run-maven-with-pepper-options:
-          module: dsm-core
-          phase: validate #NOTE: NOT RUNNING TESTS YET!
-          skip_tests: false
-          config_file_path: ''
-          skip_checkstyle: true
-          restore_m2_cache: false
-          save_m2_cache: true    #REMOVE WHEN DONE
-      - save-ddp-cache
-      - persist_to_workspace:
-          root: *root_path
-          paths:
-            - repo
+      - run: echo 'Doing dsm-compile-and-run-tests-job << pipeline.parameters.dsm >>'
+      - when:
+          condition: << pipeline.parameters.dsm >>
+          steps:
```

Comment:
The ONLY thing that changed in the added conditional statement. Bogus "run echo" step and the conditional.